### ISSUE TITLE
webpage: bg color palette has strong changes in saturation and lightness

### ISSUE THREAD

**Issue by Artoria2e5** [amark/gun#792]
The current changes in background colors can fell a bit abrupt for people as some non-hue properties of color, especially the brightness, changes as the color changes. This can be avoided by picking the colors in a more perception-based color space, such as CIE Lch:

```JS
// Run using https://npm.runkit.com/chroma-js
let chroma = require("chroma-js")
let colors = ['#ea3224', '#33cc33', '#4D79D8', '#f2b919']
// Very uneven colors
console.log(colors)
console.log(colors.map(x => chroma(x).lch()))
// Normalize C and L
let newcolors = colors.map(x => chroma(x).set('lch.l', 70).set('lch.c', 60).hex())
console.log(newcolors)
// Analyze
console.log(newcolors.map(x => chroma(x).lch()))
```

In the example above, I normalized from the colors to a l=70 and c=60, resulting in:
![image](https://user-images.githubusercontent.com/6459309/62994767-337e6400-be90-11e9-9c59-64936ff40de4.png)

Note that due to the nature of our precious sRGB color space, not every value is reachable to what we set, but the colors are already more uniform-feeling with such a change. You can of course play around with the L and C properties I set up, and optimize some other aspects such as text contrast by e.g. lowering L in the process.

**Comment by amark** [comment]
@Artoria2e5 this is brilliant!

Plus I love your analysis and proof :P

Is it necessary to have to use an additional dependency, versus just updating the CSS file with these color codes? It is just a CSS animation, and I'd rather avoid needing JS to do it.

Pardon my ignorance, I'm not very smart in these things and I admire people like you who understand all the chroma (and probably even physics & frequency) of this stuff. If I'm going for "neon" colors, what would be best? For instance, the new red seems a little dull, almost rust, compared to the previous bright vividness. However the new green, for example, does seem "nicer". The blue is decent, though seems a little too baby-ish if white text on front (font contrast!). And oooh goodness, that GOLD is PERFECT, phew you're good! 

Also note: it is 1am here and I'm in a super dark room with only my laptop light and it is set to max dim setting, and my eyes are always very tired/squinty at night, so IDK how much that may be affecting my color perception.

I'm trying to reuse https://github.com/amark/gun/blob/master/examples/style.css as the neon color stylesheet everywhere (however some sites I'm sure will need to be manually updated). But since the main CSS is here in the repo, do you want to get credit for contributing? Can we move this to a PR instead of issues?

**Comment by Artoria2e5** [comment]
you can just replace the color codes! I am fine with you just messing with the color properties a bit more and updating the CSS yourself.

**Comment by python273** [comment]
This animation is quite distracting 😬  Would be nice to provide static white/dark theme option

**Comment by amark** [comment]
@python273 http://gun.eco has a secret "batman" mode (can you find it?), maybe we can do something similar in the docs, wanna PR?

**Comment by slava-vishnyakov** [comment]
> http://gun.eco has a secret "batman" mode

@amark I seriously wish there was a non-secret "Make background normal" button instead of secret "batman" mode. I've been reading the tutorial for like 3 minutes, I can't concentrate on text and my eyes hurt already. So, now instead of reading I'm in the console trying to figure out how to stop that thing and make text readable.

**Comment by amark** [comment]
@slava-vishnyakov fair point, we're starting to build a new docs system from scratch (but could take a while, and pretty swamped on other things) I'll make sure theme switching (between blue/green/red/yellow/black/white) is available.

In the meanwhile, you'll probably prefer just reading the source of the docs, the GitHub wiki:

https://github.com/amark/gun/wiki/API

All pages on site are ^ available there too.

**Comment by slava-vishnyakov** [comment]
Thank you, @amark !
